client/src/models/ConfigModel.ts

```diff
@@ -3,18 +3,27 @@
 
 import { Theme } from '@fluentui/theme';
 
-export type PostCallSurveyType = 'msforms' | 'custom';
-interface MSFormsSurveyOptions {
+export type PostCallSurveyType = 'msforms' | 'custom' | 'onequestionpoll';
+export interface MSFormsSurveyOptions {
   surveyUrl: string;
 }
-interface CustomSurveyOptions {
+export interface CustomSurveyOptions {
   surveyUrl: string;
 }
 
+export type OneQuestionPollPromptType = 'likeorDislike' | 'rating' | 'text';
+
+export interface OneQuestionPollOptions {
+  title?: string;
+  prompt: string;
+  promptType: OneQuestionPollPromptType;
+  saveButtonText: string;
```

Comment:
we would need answer place holder also included in this right
This comes only if the option is "text input"

Reply:
Not sure why we would need a placeholder here? The placeholder is only needed in the defaultConfig right?

Reply:
answer placeholder is also a user input, but is shown only if the one question poll option is "Text input"
here is the figma: https://www.figma.com/file/zinLSNSrpQGoV6Hmbw1t1C/Virtual-visit-add-ons?node-id=424%3A72988

Reply:
Added this to the model.